design/account.go

```diff
@@ -222,6 +222,7 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	a.Attribute("company", d.String, "The company")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
 	a.Attribute("cluster", d.String, "The OpenShift API URL of the cluster where the user is provisioned to")
+	a.Attribute("featureLevel", d.String, "The level of features that the user wants to use (for unreleased features)")
```

Comment:
any reason for not using an `enum` here? Maybe to leave things flexible?

Reply:
to be able to unset it maybe :)

Reply:
yes, I think we should remain flexible here, indeed.

Reply:
Yeah... Enum was my first thought here too. But let's keep it as string because Auth should not really care what feature levels we use.